test/test_api/test_settings.py

```diff
@@ -21,3 +21,13 @@ def test_add_dynamic_mods(Script):
     result = script.goto_definitions()
     assert len(result) == 1
     assert result[0].description == 'class int'
+
+
+def test_add_bracket_after_function(Script):
+    api.settings.add_bracket_after_function = True
```

Comment:
The only problem with this is that you set it globally for all tests that run after it. Can you please use the `monkeypatch` fixture? :) Thanks!

Reply:
that should do it :)